commands/deploy.go

```diff
@@ -24,15 +26,30 @@ func (cmd *Deploy) Metadata() command_metadata.CommandMetadata {
 	return command_metadata.CommandMetadata{
 		Name:        "deploy",
 		Description: "Deploy a new brooklyn application from the supplied YAML",
-		Usage:       "BROOKLYN_NAME [ SCOPE ] deploy FILEPATH",
+		Usage:       "BROOKLYN_NAME deploy <FILEPATH|->",
```

Comment:
I'd use round brackets rather than chevrons, and put space around the separator.